### DISCUSSION TITLE
[Screencast] Customizing Checkout Steps in Solidus

### DISCUSSION THREAD

**Discussion by seand7565** [solidusio/solidus#3968] [Show and tell]
This is a question that comes up a lot here, so for the first video in this screencast series, that's what I've covered! 🎉 

https://www.youtube.com/watch?v=p_O4d9k4qxI

**Comment by jarednorman** [comment]
Nice work, Sean!

**Comment by kennyadsl** [comment]
Great stuff, thanks @seand7565!

**Comment by joyboy67** [comment]
Hello, I'm trying to add a new route 'about' to the HomeCrontroller but I have some trouble I get this error :

```
Routing Error

uninitialized constant HomeController Object.const_get(camel_cased_word) ^^^^^^^^^^ 
Rails.root: /home/computer/app
```

I have this files :

app/overrides/home_controller_decorator.rb
```
module App::HomeDecorator

	def about
	end

	Spree::HomeController.prepend self
end
```

config/route.rb 
```
Rails.application.routes.draw do
  mount Spree::Core::Engine, at: '/'
  get "/about", to: "home#about"
end
```

**Reply by jarednorman** [reply]
That isn't working because you're adding the route to your application's routes, not Solidus. You need:

```ruby
Spree::Core::Engine.routes.draw do
  get "/about", to: "home#about"
end

**Reply by joyboy67** [reply]
Thank you, it's working :)
